airflow/models/dag.py

```diff
@@ -549,6 +549,11 @@ def tasks(self, val):
     def task_ids(self):
         return list(self.task_dict.keys())
 
+    @property
+    def active_tasks(self):
+        """Return tasks in a DAG. Needed as part of determining overall DagRun outcome"""
+        return [t for t in self.tasks]
```

Comment:
Can you explain the benefit of introducing this method? In my opinion we don't need it.

Reply:
my understanding is Dag model has a static list of tasks that does not change, but dagrun could have a variable number of tasks due to db connectivity issues (ie REMOVED state) leading to potentially less tasks

Reply:
In my opinion, you can achieve the same effect without creating a new field in this class.
https://github.com/apache/airflow/pull/6954/files/9f5cf8f277365a2c7a4e3c56e86f651e5d83f618#r363543585